src/ert/gui/ertwidgets/models/storage_model.py

```diff
@@ -27,6 +29,8 @@ class _Column(IntEnum):
     2: "Type",
 }
 
+default_index = QModelIndex()
```

Comment:
Out of curiosity, is the reason for this the same as with dicts?

Reply:
So this explictly sets up the same thing as `parent=QModelIndex()`. There will only be one copy. Maybe we should just call it in the parameter? I remember there was a time where some linting rule said to just not do that.

Reply:
I will try to redo this in the `if parent is None:` way and see if mypy complains.

Reply:
Yea, this was a mistake, I should have used an Optional here as it is variant.

Reply:
Fixed it now.